src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/repository/LicenceSpecifications.kt

```diff
@@ -30,6 +30,10 @@ fun LicenceQueryObject.toSpecification(): Specification<Licence> = and(
   hasPdusIn(pdus),
   hasProbationAreaCodeIn(probationAreaCodes)
 )
+  .and { root, query, _ ->
+  root.fetch<Licence, CommunityOffenderManager>("responsibleCom", JoinType.INNER)
+  query.restriction
+}
```

Comment:
There was an inner join on the `hasResponsibleComIn` predicate below but this would only added in if the com id was passed in. 

It also didn't seem work, I think due to laziness